docs/src/components/action/NpmCopyButton.tsx

```diff
@@ -8,6 +8,7 @@ import CheckRounded from '@mui/icons-material/CheckRounded';
 
 const StyledButton = styled(Button)`
   display: inline-flex;
```

Comment:
I think we should use the same standard as in the source JavaScript style object for consistency.

Reply:
Any specific thing I should add or remove here?

Reply:
Forget about it, not worth it 😁